packages/blocks/src/api/raw-handling/html-to-blocks.js

```diff
@@ -28,6 +34,13 @@ export function htmlToBlocks( html, handler ) {
 		);
 
 		if ( ! rawTransform ) {
+			// Until the HTML block is supported in the native version, we'll parse it
+			// instead of creating the block to generate it as an unsupported block.
+			if ( Platform.isNative ) {
+				return parse(
+					`<!-- wp:html -->${ node.outerHTML }<!-- /wp:html -->`
+				);
```

Comment:
Since the HTML block is not registered in the native version, the block will be generated by simply parsing the raw HTML content of an HTML block with the HTML code provided.